### ISSUE TITLE
Problem of the exp results reported in paper

### ISSUE THREAD

**Issue by countback** [wujsAct/fine-grained-entity-typing-Hier-Denoise#1]
I have a question about the experimental results reported in the paper. For the NFETC model, the paper only reported the re-producing results, but not reported the original NFETC results (on wiki and ontonotes dataset) , in fact, which is better. So any difference between your method and the NFECT paper (like evaluation, softmax prediction & sigmoid prediction), I run the code released on https://github.com/shimaokasonse/NFGEC

**Comment by herbertchen1** [comment]
the author think NFETC use a filtered training dataset of FIGER and a tuned type tree.
Acturally, the dataset is got from the paper Attentive. Before Attentive, the exp on FIGER often have a different training set. However, it's not right to not use the result of FIGER(68.0 ACC), because the reason mentioned in this paper is not right, NFETC acturally can map the type 2 the original type system. Maybe it's better to use the same setting to beat NFETC.
Now we can talk about OntoNotes dataset, after Gillick, who remove the pronoun mention, the number of the test set is acturally 8900. I'm not sure that Ren's version or this version is also 8900.... But in this paper, it's 9600...
So "Experiments on several widely used datasets show that our
method significantly outperforms its competitors, resulting in
new state-of-the-art performance on these benchmark data" might not be right....

**Comment by herbertchen1** [comment]
Also I'm sure Abhishek's method use 128?5 test mention on BBN rather than 13,282, which I've confirm it with him on last year...

**Comment by wujsAct** [comment]
As pointed at the our paper, we just utilize the data provided by Ren et, al. https://github.com/INK-USC/AFET/tree/master/Data.  As the previous works use various training and test dataset, we report the re-produced results in this paper with the same training and test dataset. Maybe we should not repost their original experiment results when comparing all those models. Hope this has clarified all your concerns.
